user-guide/user-guide.md

```diff
@@ -10,17 +10,36 @@ description: |
 {: .fs-8 }
 
-The RAPIDS data science framework is a collection of libraries for running end-to-end data science pipelines completely on the GPU. The interaction is designed to have a familiar look and feel to working in Python, but utilizes optimized NVIDIA® CUDA® primitives and high-bandwidth GPU memory under the hood. Below are some links to help getting started with the RAPIDS libraries.
+The RAPIDS data science framework is a collection of libraries for running end-to-end data science pipelines completely on the GPU. The interaction is designed to have a familiar look and feel to working in Python, but utilizes optimized NVIDIA® CUDA® primitives and high-bandwidth GPU memory under the hood. Below are some links to help getting started with each of the individual RAPIDS libraries.
 {: .fs-6 .fw-300 .mb-8 }
 
 
+### <i class="fa-light fa-notebook"></i> Full Collection of each RAPIDS Library Example Notebooks with [RAPIDS Notebooks](https://github.com/rapidsai/notebooks){: target="_blank"}
+A GitHub repository with our introductory examples of XGBoost, cuDF, cuML, cuGraph, and more.
+{: .mb-8 }
+
+
+###  <i class="fa-light fa-notebook"></i> Extended Collection of [Community Notebooks](https://github.com/rapidsai/notebooks-contrib){: target="_blank"}
+A collection of examples and tutorials used to introduce new users to the features and capabilities of RAPIDS.
+{: .mb-8 }
+
+
+### <i class="fa-light fa-cloud"></i> Machine Learning Services Integration for [RAPIDS Cloud](https://github.com/rapidsai/cloud-ml-examples){: target="_blank"}
+A repository with example notebooks and "getting started" code samples to help you integrate RAPIDS with the hyperparameter optimization services from Azure ML, AWS Sagemaker, Google Cloud, and Databricks. 
+{: .mb-8 }
+
+
+### <i class="fa-light fa-screwdriver-wrench"></i> Tools and Guides for [RAPIDS Deployment](/deployment/stable/){: target="_blank"}
+Deployment documentation to get you up and running with RAPIDS in AWS, GCP, Azure, IBM and more. Also includes guides for HPC, HPO, Kubernetes, Dask, and more.
+{: .mb-8 }
+
 ### <i class="fa-sharp fa-solid fa-database"></i> ETL and Dataframe Processing with [cuDF](https://github.com/rapidsai/cudf){: target="_blank"}
-Start with [10 Minutes to cuDF and Dask-cuDF](/api/cudf/stable/user_guide/10min.html){: target="_blank"}. Modeled after 10 Minutes to Pandas, this is a short introduction to cuDF that is geared mainly for new users. The [cuDF User Guide](https://docs.rapids.ai/api/cudf/stable/user_guide/index.html){: target="_blank"} is generally very extensive and helpful.
+Start with the [10 Minutes to cuDF and Dask-cuDF User Guide](https://docs.rapids.ai/api/cudf/stable/user_guide/10min.html){: target="_blank"}. Modeled after 10 Minutes to Pandas, this is a short introduction to cuDF that is geared mainly for new users. The [cuDF User Guide](https://docs.rapids.ai/api/cudf/stable/user_guide/index.html){: target="_blank"} is generally very extensive and helpful.
```

Comment:
any reason these have been updated to absolute URLs vs. relative URLs?

Using absolute URLs will break these links when trying to navigate on a local server (e.g. `jekyll serve` or `hugo serve`).

Reply:
updated!